pkg/controller/dgraph/models/group.go

```diff
@@ -0,0 +1,55 @@
+package models
+
+import (
+	"time"
+
+	groups_v1 "github.com/vmware/purser/pkg/apis/groups/v1"
+	"github.com/vmware/purser/pkg/controller/dgraph"
+)
+
+// Dgraph Model Constants
+const (
+	IsGroupCRD = "isGroupCRD"
+)
+
+// GroupCRD schema in dgraph
+type GroupCRD struct {
+	dgraph.ID
+	IsGroupCRD bool      `json:"isGroupCRD,omitempty"`
+	Name       string    `json:"name,omitempty"`
+	StartTime  time.Time `json:"startTime,omitempty"`
+	EndTime    time.Time `json:"endTime,omitempty"`
+	Type       string    `json:"type,omitempty"`
+}
+
+func createGroupCRDObject(group groups_v1.Group) GroupCRD {
+	newGroup := GroupCRD{
+		Name:       group.Name,
+		IsGroupCRD: true,
+		Type:       "vmware.kuber",
```

Comment:
We need to change type from `vmware.kuber` to `vmware.purser` or just `purser`? Same change needs to be done while creating groups and subscribers.